docs/release-notes/next-ceres/announcement.md

```diff
@@ -0,0 +1,75 @@
+
+# Changes summarized
+Micro-block gas limit calculation changed: Gas limit was computed from a transaction's max gas value, now it is computed from the actually used gas.
+
+* Contracts can be called by name: Just like it is possible to spend to an AENS name, it is now possible to call a function for a name that points to a contract.
+
+* Generalized Accounts:
+  * GAMetaTx parameters (fee and gas_price) are now part of the transaction hash and thus not changeable by a rouge miner.
+  * GAAttachTx can only be used by a fresh account.
+* AENS:
+  * Adjusted auction parameters - auctions are generally made shorter.
+  * Adding raw data pointers.
+  * Adding wildcard delegation signature, delegating control for all current, and future, names for an account to a particular contract.
+  * PreClaimTx is made optional.
+* FATE:
+  * Delegation signatures have been made more structured.
+  * Support for dynamic-sized binary arrays (type bytes()) is added.
+  * Built-in bitwise operations have been added.
+  * Crypto.poseidon - a SNARK-friendly hash function is added.
+* Utility functions Chain.network_id, Address.to_bytes, and Int.mulmod are added.
+* AENSv2 is introduced to support the raw data pointers.
+* BRI - the Block Reward Initiative has been made configurable. The BRI was introduced in Fortuna and hasn’t changed since. In Ceres it has been made a proper part of the configuration, and the mechanism has been updated to allow changing it as part of a protocol upgrade. Values for mainnet and testnet remain hardcoded.
+
+# Changes detailed
+
+These changes are provided by the PR's:
+https://github.com/aeternity/aeternity/pulls?q=is%3Apr+label%3Abreaking%2Fconsensus+
+
+## Micro-block gas limit
+
+Instead of using max allowed gas when packing micro-blocks we use actual used gas. This allows for more transactions to be packed into a block, increasing throughput and also stopping (accidental) denial of resources when overestimating contract calls, etc. The downside of this is that the transaction has to be applied before its gas usage is used. This puts a little bit of extra burden on the block producer, but, thankfully, block verification is not computationally more expensive.
+
+## Contracts called by name
+Contracts can be called by name. Just like it is possible to spend to an AENS name, it is now possible to call a function for a name that points to a contract. The key contract_pubkey of the name is resolved while running the transaction. Technically, this has consequences for the contract call structure
+(aect_call) that gets a new field with the contract call identifier. To be able to find the call result without doing name resolution calls are stored relative to the id rather than the contract pubkey.
```

Comment:
"calls are stored relative to the id rather than the contract pubkey" can I get more information on what exactly happens here, and just to be sure I fully understand, also the motivation behind this?

Reply:
Previously a contract call had a field "contract_id" (don't remember exact name). We could have kept using that, but then in order to find the call (and with call here we mean the aect_call structure stored in the contract call tree) you would have to do a name resolution - at the right height, etc. Instead we opted for adding a field "ct_call_id" which is either `undefined` when calling a contract directly or `namehash` if called through a name. 

The result of this gymnastics is that you can find the call without doing a name resolution.